Core/include/Acts/EventData/detail/DifferenceCalculator.hpp

```diff
@@ -0,0 +1,60 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2016-2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/EventData/detail/ParameterTraits.hpp"
+
+#include <utility>
+
+#include <Eigen/Core>
+
+namespace Acts {
+namespace detail {
+
+/// Compute the minimal difference between parameters within the nominal range.
+///
+/// @tparam indices_t Parameter indices enum type
+/// @tparam kParameters Parameter indices pack that defines the used parameter
+template <typename indices_t, indices_t... kParameters>
+struct DifferenceCalculator {
+  using StorageSequence = std::make_index_sequence<sizeof...(kParameters)>;
+
+  /// Run the difference calculation.
+  ///
+  /// @param rhs Right-hand-side input vector
+  /// @param lhs Left-hand-side input vector
+  /// @return minimal, corrected (lhs - rhs) difference within the nominal range
+  template <typename lhs_t, typename rhs_t>
+  static constexpr auto run(const Eigen::MatrixBase<lhs_t>& lhs,
```

Comment:
This is also not actually `constexpr`. I know you say it doesn't hurt to put it there, but I think it suggests this was a `constexpr` function, which it is not I believe.

Reply:
I will remove it.

Reply:
Removed.